src/python/pants/bin/pants_runner.py

```diff
@@ -57,9 +58,15 @@ def run(self):
     bootstrap_options = options_bootstrapper.bootstrap_options
     global_bootstrap_options = bootstrap_options.for_global_scope()
 
+    # N.B. Inlining this import speeds up the python thin client run by about 100ms.
```

Comment:
I believe that moving this here undoes the optimization referenced by the comment, so should try to avoid that.